src/register-org/components/payment-by-account/payment-by-account.component.html

```diff
@@ -6,13 +6,11 @@
       <span class="govuk-caption-xl">Register an organisation</span>
       <form [formGroup]="pbaFormGroup">
         <div class="govuk-form-group">
-          <fieldset class="govuk-fieldset" aria-describedby="changed-name-hint">
-            <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
-              <h1 class="govuk-fieldset__heading">
-                Does your organisation have a payment by account number?
-              </h1>
+          <fieldset class="govuk-fieldset">
```

Comment:
@johnbenjamin-hmcts - for accessibility, add `aria-describedby="pba-hint"`

Reply:
Thank you @Daniel-Lam - I have updated the code now.